### PR TITLE
Standalone

### PR DESCRIPTION
**Pull request by brumik**
* have backend running
* no proxy needed
* make sure you run `npm ci` after checkout
* run `npm start`
* visit: http://localhost:1337/beta/ansible/insights/automation-calculator
* Login: admin, admin

### CODE REVIEW

**Comment by brumik** [comment]
PR merged, you can try the following in @redhat-cloud-services/frontend-components-config@4.3.0 :
  registry: [
    ({ app }) => app.get('(/beta)?/config/main.yml', (_req, res) => res.sendFile(resolve(__dirname, './main.yml')))
  ]

**Comment by h-kataria** [comment]
@brumik found one issue after I applied some filters and refresh from the browser, it blows up. Try this URL: http://localhost:1337/beta/ansible/insights/automation-calculator?clusters.sort_options=total_cluster_count&clusters.sort_order=asc

I see following error in console
```
Uncaught TypeError: document.getElementById(...) is null
    js entry-dev.js:1
    Webpack 5
entry-dev.js:1
    js entry-dev.js:1
    Webpack 5
```

File: cypress/integration/Dashboard.spec.js [hunk 1]
```diff
@@ -1,133 +1,140 @@
...
-
-        // click it and wait for the jobexplorer page to load ...
-        cy.get(appid).find('rect').eq(barid).click({ waitForAnimations: true });
-        cy.screenshot('clusters-bar-' + barid + '-jobexplorer-details.png', { capture: 'fullPage' });
-
-        // go back to the clusters page ...
-        cy.visit(dashboardUrl);
-    }
-
+        cy.get('button[aria-label="Close"]').click({ waitForAnimations: true });
```
Comment (h-kataria) [inline cypress/integration/Dashboard.spec.js:36]: @brumik do we need to keep all this commented out lines?

File: cypress/integration/Dashboard.spec.js [hunk 2]
```diff
@@ -1,133 +1,140 @@
...
     requires setting it on the prototype level of the native input HTML element itself
-*/
-const setDate = (input, value) => {
-    const nativeInputValueSetter = Object.getOwnPropertyDescriptor(window.HTMLInputElement.prototype, 'value').set;
-    nativeInputValueSetter.call(input, value);
-
-    const event = new Event('input', { bubbles: true });
-    input.dispatchEvent(event);
-};
+// */
```
Comment (h-kataria) [inline cypress/integration/Dashboard.spec.js:13]: i think this can be deleted

**Review by h-kataria — APPROVED** [review]
other than the minor comments on the commented out lines, this looks good to me, tested in UI works great. Ready to be merged.

File: cypress/integration/Dashboard.spec.js (hunk 1 above)
Comment (brumik) [reply cypress/integration/Dashboard.spec.js:36]: Aparna's PR handles the tests. I just modified the minimum here for testing reasons.

File: cypress/integration/Dashboard.spec.js (hunk 2 above)
Comment (brumik) [reply cypress/integration/Dashboard.spec.js:13]: Same as above

**Pull request merged**